database/login.js

```diff
@@ -0,0 +1,15 @@
+const pg = require('./index.js').pg;
+
+module.exports = {
+  checkPasswordAtUsername: (username, callback) => {
+    pg.table('users')
+      .where({username: username})
+      .select('id', 'password')
+      .then((result) => {
+        callback(result);
+      });
+
```

Comment:
Is there a catch statement? Promises (IMO) should always have a catch statement. In this case, if the database were down or something, I think the server would crash. Better to catch it and output an error message.